renovate.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "config:base"
```

Comment:
The base config is here: https://renovatebot.com/docs/presets-config/#configbase